.gitignore

```diff
@@ -63,3 +63,10 @@ yarn-debug.log*
 
 # Ignore Font files in accordance to Licensing Agreement
 next-frontend/src/app/(wca)/fonts/TTNormsPro/*
+
+# Playwright
+node_modules/
```

Comment:
This is not Playwright-related. Why are you adding this here? It should already be gitignored through some other file(s)